server/services/teamsSummaryCountService.ts

```diff
@@ -168,12 +168,40 @@ export default class TeamsSummaryCountService {
 
       const componentNamesSet = new Set(validComponents)
 
+      // Track summary-based counts for comparison
+      let summaryFixedCritical = 0
+      let summaryFixedHigh = 0
+      let summaryUnfixedCritical = 0
+      let summaryUnfixedHigh = 0
+
       const counts = trivyScans.reduce(
         (accumulator, scan) => {
           if (!componentNamesSet.has(formatMonitorName(scan.name))) {
             return accumulator
           }
 
+          // Log summary-based counts (like Trivy table uses)
+          const summary = scan?.scan_summary?.summary
+          if (summary) {
```

Comment:
You can either ditch this `if` statement because you're doing `summary?` or ditch the `?` on the `summary?` statement. Personally I'd just ditch the `if`